src/lava/proc/dense/process.py

```diff
@@ -8,7 +8,7 @@
 
 
 class Dense(AbstractProcess):
-    """Dense connections between neurons. Realizes the following abstract
+    """Dense connections between neurons. Realizes the      following abstract
```

Comment:
We probably have to rethink this again. The documentation below would probably be surprising to the reader because it  reveals knowledge about what's to come in terms of implementation. A Process was meant to be HW agnostic.

This is not ideal:
weights = weights * (2 ** w_scale)
w_scale =  8 - num_weight_bits + weight_exp + isMixed()
In NxSDK so far, we have treated 'weights' as the invariant set of parameters with dynamic range from -256..255. num_wgt_bits was just used to scale the weight increment behind the scenes.
I guess for now it's fine. But we should also revisit this.

Reply:
Sounds good. Yes, this implementation was chosen to mirror the implementation in the prototype of Lava, but we should perhaps not expose so much of this implementation at the level of the Process. I have added a ToDo so we remember to address this in a future issue.